src/Components/UpComingQuiz/UpComingQuiz.tsx

```diff
@@ -0,0 +1,77 @@
+import psyology from "../../assets/443de449f9295558ab8120f068b0b58b.png";
+import intro from "../../assets/Quiz img (2).png";
+import { Link } from "react-router-dom";
+
+export default function UpComingQuiz() {
```

Comment:
This component holds all upcoming quizes, so we should make it plural to indicate that , also upcoming is a single word not Up  - Coming
so you can consider renaming the component to `UpcomingQuizes`